### PR TITLE
Added dvpe-netfilter helm chart for update iptables on kubelet nodes

### PR DESCRIPTION
**Pull request by cd-at-iteratec**
Signed-off-by: Cliff Dornig <cliff.dornig@iteratec.com>

### CODE REVIEW

File: charts/dvpe-netfilter/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2021-05-05
```
Comment (mithie) [inline charts/dvpe-netfilter/CHANGELOG.md:10]: please remove the release date. we committed on using the version number only

File: charts/dvpe-netfilter/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+appVersion: "0.1"
+description: Helm chart for set netfilter rules on Kubelet node
```
Comment (mithie) [inline charts/dvpe-netfilter/Chart.yaml:3]: typo: "for setting"

File: charts/dvpe-netfilter/README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+# dvpe-netfilter
+
+![Version: 0.1](https://img.shields.io/badge/Version-0.1-informational?style=flat-square)
+
+Helm chart for set netfilter rules on Kubelet node
```
Comment (mithie) [inline charts/dvpe-netfilter/README.md:5]: typo: for setting

File: charts/dvpe-netfilter/README.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Installation
+Installs dvpe-helm netfilter injector on an existing Kubernetes cluster.
+
+**Note**: The netfilter injector needs to run as root with NET_ADMIN capabilties and use hostNetwork in namespace kube-system.
```
Comment (mithie) [inline charts/dvpe-netfilter/README.md:10]: I would add this additional sentence: 

`When Pod Security Policies are used in your cluster, then they need to reflect those requirements.`

File: charts/dvpe-netfilter/templates/configmap.yaml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+metadata:
+  name: update-iptables-script
+  namespace: kube-system
+data:
+  update-iptables.sh: |
+    #!/bin/bash
+    apk update
+    apk add iptables
+    {{- range $if := .Values.netfilter.blocked.interfaces }}
+    {{- range $ip := $.Values.netfilter.blocked.ipv4 }}
```
Comment (mithie) [inline charts/dvpe-netfilter/templates/configmap.yaml:13]: remove $ from $.Values.netfilter.blocked.ipv4

**Review by mithie — CHANGES_REQUESTED** [review]
great job! only found some minor issues

File: charts/dvpe-netfilter/templates/configmap.yaml (hunk 5 above)
Comment (cd-at-iteratec) [reply charts/dvpe-netfilter/templates/configmap.yaml:13]: Hi @mithie ,

the $ is required to work in helm correctly.

**Comment by cd-at-iteratec** [comment]
@mithie : Thanks for reviewing.
I fixed the typos and recommendations. 
Could please review again ?

Thanks,
Cliff

**Review by mithie — APPROVED** [review]

**Pull request merged**